### DISCUSSION TITLE
Custom FQDN for Receiver Node

### DISCUSSION THREAD

**Discussion by senatesan** [Security-Onion-Solutions/securityonion#11052] [2.4] [answered]
The following Advanced Grid Configuration in the SOC did not update the SAN in the receiver node's `elasticfleet-logstash.crt` although it did update this .crt on the manager.

`elasticfleet > config > server > custom_fqdn`

If the receiver is acting as a backup/load balancer logstash output for fleet, certificate errors on the endpoints occur:

```
[elastic_agent.filebeat][error] Failed to connect to failover(backoff(async(tcp://<custom_fqdn1>:5055)),backoff(async(tcp://<custom_fqdn2>:5055))): x509: certificate is valid for <receiver_hostname>, not <custom_fqdn2>
```

**Comment by defensivedepth** [comment]
What version of 2.4 are you running?

**Reply by senatesan** [reply]
I had the issue with 2.4.10 so I did a clean install of 2.4.10 Hotfix [20230821]

**Security Onion version**
2.4.10 Hotfix [20230821]

 **Cloud deployment or on-prem**
On-prem with internet access

**Install from our Security Onion ISO image or did you perform a network installation**
ISO

**How many nodes**
6

**Hardware specs of each of those nodes**
8 CPU, 16GB RAM, 500 GB HDD, 1Gbps NICs

**How are each of those nodes configured**
Manager with one receiver, one fleet, two search, one forward node

 **Issues monitoring network traffic**
No

**Does so-status show all services running**
Yes

**Failures when you run sudo salt-call state.highstate**
No

**Does the SOC Grid page show any failures**
No

**Issue**

On the manager running `openssl x509 -in /etc/pki/elasticfleet-logstash.crt -text -noout` shows the values of `custom_fqdn` in the `X509v3 Subject Alternative Name` `DNS` section

However, the same command ran on the receiver node does not show the values of `custom_fqdn` in the` X509v3 Subject Alternative Name` `DNS` section

**Comment by defensivedepth** [comment, marked as answer]
Receiver nodes are specifically for Grid Nodes and `allows for events to continue to be processed by search nodes in the event the manager node is offline.`

They are not meant to be used by non-Grid Elastic Agents (ie Windows laptops etc).

As such, they do not need the Custom FQDN, as Grid Nodes will connect to the Receiver using the Receiver's hostname, not the Custom FQDN.

In other words, this is expected behavior.

**Reply by senatesan** [reply]
I misunderstood the documentation. I suppose the Fleet node is the one to be used to load balance events between the Manager node from non-grid elastic agents?

**Reply by defensivedepth** [reply]
Yes that is correct!
